### PR TITLE
fix(common/models): proper RTL quote ordering

### PR DESCRIPTION
**Pull request by jahorton**
Fixes the output ordering of quotation-style punctuation characters for RTL languages.

### CODE REVIEW

File: common/models/templates/test/test-quote-behavior.js [hunk 1]
```diff
@@ -31,7 +31,18 @@ describe('Quote behaviors', function() {
     });
 
     it.skip('RTL', function() {
```
Comment (darcywong00) [inline common/models/templates/test/test-quote-behavior.js:33]: Is this test getting skipped?

Comment (jahorton) [reply common/models/templates/test/test-quote-behavior.js:33]: I left it as skipped b/c of the comment at the bottom of the `function`.  Basically, the test doesn't read intuitively, and functionally it feels about the same as the baseline LTR version.  I can un-skip it though, if y'all prefer.

**Comment by mcdurdin** [comment]
Can you give some examples of the model definition and corresponding screenshot of the results?

**Comment by darcywong00** [comment]
> Can you give some examples of the model definition and corresponding screenshot of the results?

imho a test page that showed the quoted suggestions might be helpful. You could use the gilaki keyboard and sil.glk-arab.ptwl1 lexical-model for displaying RTL.

**Comment by jahorton** [comment]
> Can you give some examples of the model definition and corresponding screenshot of the results?

That depends on how many RTL models we have available.  I'm not exactly an expert on RTL-script languages, after all, so unless we want theoretical, constructed examples... we'll want actual models to use.

Darcy's noted one model above... though even then, that model isn't actually marked as RTL.

**Comment by jahorton** [comment]
So... it looks like this will be stalled until some good examples are available, and I'm at a bit of a loss in that department.

**Comment by darcywong00** [comment]
> . though even then, that model isn't actually marked as RTL.

as I recall, RTL is tracked at the keyboard level, not lexical-models.
I think a test page with gilaki keyboard and sil.glk-arab.ptwl1 should be able to demonstrate RTL

**Comment by jahorton** [comment]
> 
> 
> > . though even then, that model isn't actually marked as RTL.
> 
> as I recall, RTL is tracked at the keyboard level, not lexical-models.
> I think a test page with gilaki keyboard and sil.glk-arab.ptwl1 should be able to demonstrate RTL

It's not an either-or.  It's both:

https://github.com/keymanapp/keyman/blob/36075815aad524f5d852fb30c10b04ca1ba3dc07/common/models/types/index.d.ts#L559-L564

We need `.isRTL` as part of the punctuation spec.  This has been there for a while - since at least 13.0, and possibly before that.  Here's the same code on `stable-13.0`.

https://github.com/keymanapp/keyman/blob/1e844297b249313399c3d5a865e845cbf80f62a6/common/lexical-model-types/index.d.ts#L365-L370

**Comment by mcdurdin** [comment]
Can you take that model that @darcywong00 mentioned, add the isRTL property to it, and test with that?

**Comment by jahorton** [comment]
Please keep in mind, I do _not_ know what I'm doing with that keyboard / lexical-model pair.  That said...  I went ahead and made [lexical-models/#106](https://github.com/keymanapp/lexical-models/pull/106), then used it for this:

Screen Shot 2020-11-26 at 10 57 28 AM

(This branch doesn't have the recent OSK-font adjustments on it, so please forgive that aspect of this screenshot.)

_**That**_ said, note that `«` and `»` glyph-swap when in RTL mode.
- That is, the glyph on the right-hand side (the RTL 'open' quote) is `»`, but is represented internally as the character code for `«`.

-----

Let me know if that's satisfactory.  If not, please specify what else you might be looking for.

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**